modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -77,27 +77,27 @@ The `--copy-network` option only copies networking configuration found under `/e
 
 The disk partitions are created on {product-title} cluster nodes during the {op-system-first} installation. Each {op-system} node of a particular architecture uses the same partition layout, unless the default partitioning configuration is overridden. During the {op-system} installation, the size of the root file system is increased to use the remaining available space on the target device.
 
+Kubernetes supports monitoring `/var/lib/containers` when mounted in the same partition as `/var` or a separate partition. 
```

Comment:
Remove that sentence; it provides no value / information. Instead:

```
Red Hat OpenShift Container Platform monitors two different filesystem identifiers:
* nodefs - the filesystem which contains /var/lib/kubelet
* imagefs - the filesystem which contains /var/lib/containers

With the default partition scheme, nodefs and imagefs monitor the same filesystem (i.e., the root filesystem `/`).
```

Reply:
Thanks, Andreas. @djuran , would you be OK with me removing this sentence and replacing it with Andreas' recommendation?

Reply:
Yes, @andreaskaris suggestion looks correct

Reply:
Thanks, David.

@andreaskaris . Updated to: 

> Red Hat {product-title} monitors the following two filesystem identifiers:

* `nodefs`: the filesystem that contains `/var/lib/kubelet`
* `imagefs`: the filesystem that contains `/var/lib/containers`

For the default partition scheme, `nodefs` and `imagefs` monitor the same root filesystem, `/`.